From 026e3d3f0b016ba4bda7f0a307e44e3c61ee7300 Mon Sep 17 00:00:00 2001
From: Jiali Chen <chenjiali@radxa.com>
Date: Tue, 31 Mar 2026 10:36:10 +0000
Subject: [PATCH 140/268] drm/bridge: tc358762: switch from sync pulse to burst
 mode

Change DSI mode from MIPI_DSI_MODE_VIDEO_SYNC_PULSE to
MIPI_DSI_MODE_VIDEO_BURST for improved display compatibility.

Signed-off-by: Jiali Chen <chenjiali@radxa.com>
---
 drivers/gpu/drm/bridge/tc358762.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/gpu/drm/bridge/tc358762.c
+++ b/drivers/gpu/drm/bridge/tc358762.c
@@ -278,7 +278,7 @@ static int tc358762_probe(struct mipi_ds
 	/* TODO: Find out how to get dual-lane mode working */
 	dsi->lanes = 1;
 	dsi->format = MIPI_DSI_FMT_RGB888;
-	dsi->mode_flags = MIPI_DSI_MODE_VIDEO | MIPI_DSI_MODE_VIDEO_SYNC_PULSE |
+	dsi->mode_flags = MIPI_DSI_MODE_VIDEO | MIPI_DSI_MODE_VIDEO_BURST |
 			  MIPI_DSI_MODE_LPM | MIPI_DSI_MODE_VIDEO_HSE;
 
 	ret = tc358762_parse_dt(ctx);
